03_letters/lokili/main_test.go

```diff
@@ -0,0 +1,46 @@
+package main
+
+import (
+	"bytes"
+	"testing"
+
+	"github.com/stretchr/testify/require"
+)
+
+func TestMainOutput(t *testing.T) {
+	r := require.New(t)
+	// Given
+	var buf bytes.Buffer
+	out = &buf
+
+	// When
+	main()
+
+	// Then
+	expected := `a:2
+b:1`
+	actual := buf.String()
+	r.Equalf(expected, actual, "Unexpected output in main()")
+}
+
+func TestLetters(t *testing.T) {
+	r := require.New(t)
+	letterMap := letters("lokili")
+	expected := map[rune]int{'i': 2, 'k': 1, 'l': 2, 'o': 1}
+	r.Equalf(expected, letterMap, "Unexpected output in main()")
```

Comment:
Update the comment `"Unexpected output in main()"` to match the method that is being tested.

Reply:
I wouldn't have a comment at all. The developer will invariably have to come back to the code to investigate, which provides all the context they need. If, say, this was inside a for-loop, it might be worthwhile to output the loop index.